src/components/Auth/Auth.js

```diff
@@ -22,6 +22,14 @@ class Auth extends PureComponent {
 
   static defaultProps = defaultProps
 
+  componentWillMount() {
```

Comment:
`componentWillMount` is deprecated, move it to didMount or constructer instead

Reply:
@jarindr fixed :)